Sources/Action/Action.swift

```diff
@@ -110,6 +110,7 @@ public final class Action<Input, Element> {
         executing = Observable
             .of(executionStart.map { _ in true }, executionEnd.map { _ in false })
             .merge()
+            .shareReplay(1)
             .startWith(false)
```

Comment:
I think `shareReplay(1)` should be placed after `startWith(false)` so that late subscribers can received the latest value as an initial value, not fixed `false`.

```swift
executing = Observable
    .of(executionStart.map { _ in true }, executionEnd.map { _ in false })
    .merge()
    .startWith(false)
    .shareReplay(1)
```